contracts/access/manager/IAccessManager.sol

```diff
@@ -107,9 +107,6 @@ interface IAccessManager {
      * NOTE: The IAuthority interface does not include the `uint32` delay. This is an extension of that interface that
      * is backward compatible. Some contracts may thus ignore the second return argument. In that case they will fail
      * to identify the indirect workflow, and will consider calls that require a delay to be forbidden.
-     *
-     * NOTE: This function does not report the permissions of this manager itself. These are defined by the
-     * {_canCallSelf} function instead.
```

Comment:
Similarly, I think it's valuable to reference AccessManager's own permissions rather than the private function. Let's do:

```suggestion

     * NOTE: This function does not report the permissions of this manager itself. These are defined by the
     * {AccessManager} documentation.
```